src/layouts/BaseLayout/BaseLayout.styles.ts

```diff
@@ -1,13 +1,15 @@
 import { styled } from '@mui/system';
 import { Box } from '@mui/material';
 
+import { LEFT_BAR_WIDTH } from 'consts';
+
 export const StyledBaseLayout = styled(Box)`
   height: 100vh;
   display: flex;
 `;
 
 export const StyledCol = styled(Box)`
-  width: 100%;
+  width: calc(100% - ${LEFT_BAR_WIDTH});
```

Comment:
Why need calc(100% - ${LEFT_BAR_WIDTH});

Reply:
Because it's wrongly count width in some cases